### PR TITLE
initial commit for container insights docs

### PR DESCRIPTION
**Pull request by parkand1**
Initial commit for container insights docs - team let me know if this is what we're looking for here

### CODE REVIEW

File: docs/addons/container-insights.md [hunk 1]
```diff
@@ -21,3 +28,121 @@ new EksBlueprint(app, 'my-stack-name', addOns, [], {
...
+   2. Audit (audit) – kube-apiserver-audit-<nnn...>
+   3. Authenticator (authenticator) – authenticator-<nnn...>
+   4. Controller manager (controllerManager) – kube-controller-manager-<nnn...>
+   5. Scheduler (scheduler) – kube-scheduler-<nnn...>
+
+Next in the console, click on Log groups under Logs. 
+
+You will see under log streams all the log streams from your Amazon EKS control plane. 
+
+(insert image)
```
Comment (shapirov103) [inline docs/addons/container-insights.md:112]: Let's get the image and reference it here.

File: docs/addons/container-insights.md [hunk 2]
```diff
@@ -21,3 +28,121 @@ new EksBlueprint(app, 'my-stack-name', addOns, [], {
...
+
+## View workload level logs
+
+In order to view workload level logs follow these steps after browsing to the CloudWatch Logs Insights console
+
+In the navigation pane, choose Insights.
+
+Near the top of the screen is the query editor. When you first open CloudWatch Logs Insights, this box contains a default query that returns the 20 most recent log events.
+
+In the box above the query editor, select one of the Container Insights log groups to query. For the following example queries to work, the log group name must end with performance.
```
Comment (shapirov103) [inline docs/addons/container-insights.md:123]: Which example?

File: docs/addons/container-insights.md [hunk 3]
```diff
@@ -21,3 +28,121 @@ new EksBlueprint(app, 'my-stack-name', addOns, [], {
...
+STATS avg(number_of_container_restarts) as avg_number_of_container_restarts by PodName
+`SORT avg_number_of_container_restarts DESC`
+This query displays a list of your pods, sorted by average number of container restarts.
+
+If you want to try another query, you can use include fields in the list at the right of the screen. For more information about query syntax, see CloudWatch Logs Insights Query Syntax.
+
+## View metrics for cluster and workloads
+Under Performance monitoring, the Container Insights dashboard allows you to hone in on both cluster and workload metrics. After selecting EKS Pods and Clusters, you will see that the dashboard provides CPU and memory utilization along with other important metrics such as network performance. 
+
+## View containers via that container map in container insights.
```
Comment (shapirov103) [inline docs/addons/container-insights.md:144]: Let's move this section to the top with the picture. It provides high level insights. The details then follow.

File: docs/addons/container-insights.md [hunk 4]
```diff
@@ -21,3 +28,121 @@ new EksBlueprint(app, 'my-stack-name', addOns, [], {
...
+
+In order to view a map of all of your containers running inside your cluster, click on `View your container map` in the Container Insights tab. You will then see a map of all of your namespaces and their associated pods and services.
+
+(insert image)
```
Comment (shapirov103) [inline docs/addons/container-insights.md:148]: Let's add picture.

File: docs/addons/container-insights.md [hunk 5]
```diff
@@ -21,3 +28,121 @@ new EksBlueprint(app, 'my-stack-name', addOns, [], {
...
+NAME                           READY   STATUS    RESTARTS   AGE
+pod/cloudwatch-agent-k8wxl     1/1     Running   0          105s
+pod/fluentd-cloudwatch-78zv4   1/1     Running   0          105s
+
+NAME                                DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+daemonset.apps/cloudwatch-agent     1         1         1       1            1           <none>          107s
+daemonset.apps/fluentd-cloudwatch   1         1         1       1            1           <none>          106s
+```
+
+To enable or disable control plane logs with the console, please run the following command in your terminal - `aws eks update-cluster-config \
```
Comment (kcoleman731) [inline docs/addons/container-insights.md:46]: we should add this by default via the addon. And perhaps a option to disable via addon props.

**Review by kcoleman731 — COMMENTED** [review]
Looking great.

File: docs/addons/container-insights.md (hunk 5 above)
Comment (kcoleman731) [reply docs/addons/container-insights.md:46]: Not needed for this PR though. I have filed a ticket. https://github.com/aws-quickstart/quickstart-ssp-amazon-eks/issues/79

Comment (parkand1) [reply docs/addons/container-insights.md:46]: +1 sounds good

File: docs/addons/container-insights.md (hunk 1 above)
Comment (parkand1) [reply docs/addons/container-insights.md:112]: Added

File: docs/addons/container-insights.md (hunk 3 above)
Comment (parkand1) [reply docs/addons/container-insights.md:144]: Done

File: docs/addons/container-insights.md (hunk 4 above)
Comment (parkand1) [reply docs/addons/container-insights.md:148]: Done

File: docs/addons/container-insights.md (hunk 2 above)
Comment (parkand1) [reply docs/addons/container-insights.md:123]: added examples for both

**Review by kcoleman731 — APPROVED** [review]

**Pull request merged**